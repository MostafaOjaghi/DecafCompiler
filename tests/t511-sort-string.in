Alieh
Ali
Zahra
Mammad
Hamed
Hamid
Kazem
END