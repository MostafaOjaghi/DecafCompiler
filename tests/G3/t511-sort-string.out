Please enter the strings (max count: 100, enter END to end sooner):
After sort:
Ali
Alieh
Hamed
Hamid
Kazem
Mammad
Zahra